tools/lsp/preview.rs

```diff
@@ -61,6 +61,7 @@ type SourceCodeCache = HashMap<Url, (SourceFileVersion, String)>;
 #[derive(Default)]
 struct ContentCache {
     source_code: SourceCodeCache,
+    resources: HashSet<Url>,
     dependency: HashSet<Url>,
```

Comment:
I suppose this field should be called `dependencies` since it's passed to `fn extract_resources(dependencies: ...)` :)

Reply:
Not in this patch, but yes, that should be dependencies.